src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier color = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        String randomColor = String.valueOf(color.getRandomColor());

```

Comment:
do not forget about Common mistakes:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#common-mistakes
For enams, instead of the String.valueOf() method, we use .name()